arts-university-bournemouth.csl

```diff
@@ -174,7 +174,7 @@
       <key macro="year-date" sort="descending"/>
       <key macro="author-short"/>
     </sort>
-    <layout prefix="(" suffix=")" delimiter="; ">
+    <layout prefix="(" suffix=")." delimiter="; ">
```

Comment:
We don't want to have the full stop after the ')' in our in-text citation, only in our full reference. I just didn't know which ')' was in the full reference (i can really read csl code), so i just put one after all of them and planned to figure it out after we had tested it. I'll retry...